main/Accordion/AccordionItem.tsx

```diff
@@ -92,86 +105,87 @@ const AccordionItem: FC<Props> = (props) => {
     setItemVisible(!isItemVisible);
   };
 
-  useEffect((): void => {
+  useCallback((): void => {
```

Comment:
Is there any reason to change this line?

Reply:
@DevYuns I am curious either.